config.w32

```diff
@@ -68,6 +69,9 @@ if (PHP_MONGODB != "no") {
     /I" + configure_module_dirname + "/src/libmongoc/src/libbson/src \
     /I" + configure_module_dirname + "/src/libmongoc/src/libbson/src/jsonsl \
     /I" + configure_module_dirname + "/src/libmongoc/src/libmongoc/src \
+    /I" + configure_module_dirname + "/src/libmongocrypt/src \
+    /I" + configure_module_dirname + "/src/libmongocrypt/kms-message/src \
+    /I" + configure_module_dirname + "/src/libmongocrypt-compat \
```

Comment:
Should these three includes be wrapped in a `(PHP_CLIENT_SIDE_ENCRYPTION != "no")` conditional?